save-backend/src/test/kotlin/org/cqfn/save/backend/DownloadFilesTest.kt

```diff
@@ -1,25 +1,21 @@
 package org.cqfn.save.backend
 
-import org.cqfn.save.backend.repository.ProjectRepository
-
 import org.junit.jupiter.api.Test
 import org.springframework.beans.factory.annotation.Autowired
-import org.springframework.boot.test.autoconfigure.web.reactive.WebFluxTest
-import org.springframework.boot.test.mock.mockito.MockBean
+import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient
+import org.springframework.boot.test.context.SpringBootTest
 import org.springframework.core.io.ByteArrayResource
 import org.springframework.http.client.MultipartBodyBuilder
+import org.springframework.test.context.ActiveProfiles
 import org.springframework.test.web.reactive.server.WebTestClient
 import org.springframework.web.reactive.function.BodyInserters
 
 import kotlin.io.path.ExperimentalPathApi
 
-@WebFluxTest
-class DownloadFilesTest {
-    @MockBean
-    var repository: ProjectRepository? = null
-
-    @Autowired
-    lateinit var webClient: WebTestClient
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
```

Comment:
`SpringBootTest` is more about e2e testing. Will combination of `@ExtendWith(SpringExtension.class)`,
`@WebFluxTest` and `@Import` with necessary components be more appropriate here?

Reply:
Nevermind, let's use `SpringBootTest`, but with testcontainers, which are already in master.